app/assets/javascripts/fed_ax_api.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Some devs are pretty adamant about not committing empty javascript, or css files. I didn't stop doing this until I was in my internship but figured I'd pass along the info.